product_docs/docs/postgis/3.1.4/01_release_notes.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: "Release Notes"
+---
+EDB PostGIS is a PostgreSQL extension that allows you to store Geographic Information Systems (GIS) objects in an Advanced Server database.
+
+New features, enhancements, bug fixes, and other changes in PostGIS 3.1.4 include:
+
+| Type            | Description                                                                                                                                      |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Upstream Merges | Merged with community PostGIS 3.1.4. See [Community Release Notes](https://postgis.net/docs/manual-3.1/release_notes.html#idm41899) for details. |
```

Comment:
```suggestion
| Upstream Merges | Merged with community PostGIS 3.1.4. See the Community [Release Notes](https://postgis.net/docs/manual-3.1/release_notes.html#idm41899) for details. |
```

Reply:
Alternatively, we could have 3.1.4 be a link to the release notes, but I think I prefer the way you have done it.
Question, why are we listing those two other bugs? They seem to be listed in the community release notes so I'm not sure why we need to add them.

Reply:
I added those bugs from Release Annoucement doc